tests/sentry/issues/test_producer.py

```diff
@@ -75,127 +77,167 @@ def test_with_status_change(self):
             },
             project_id=self.project.id,
         )
-        group = event.group
-        assert group
-        group_hash = GroupHash.objects.filter(group=group, project=self.project).first()
-        initial_status = group.status
-        initial_substatus = group.substatus
+        self.group = self.event.group
+        assert self.group
+        self.group_hash = GroupHash.objects.filter(group=self.group, project=self.project).first()
+        self.initial_status = self.group.status
+        self.initial_substatus = self.group.substatus
 
-        status_change_resolve = OccurrenceStatusChange(
-            fingerprint=[group_hash.hash],
-            project_id=group.project_id,
-            new_status=initial_status,
-            new_substatus=initial_substatus,
+    def test_with_invalid_payloads(self) -> None:
+        with pytest.raises(ValueError):
+            produce_occurrence_to_kafka(
+                payload_type=PayloadType.OCCURRENCE,
+            )
+
+            produce_occurrence_to_kafka(
+                payload_type=PayloadType.STATUS_CHANGE,
```

Comment:
Why does this raise a value error?

Reply:
The status change payload type needs to be accompanied with the StatusChangeOccurrence object - added comments to the tests to explain more.